ehitchhiking-ui/src/app/app.module.ts

```diff
@@ -7,18 +7,20 @@ import {FormsModule} from '@angular/forms';
 import {MainScreenModule} from './pages/main-screen/main-screen.module';
 import {MenuModule} from './shared/components/menu/menu.module';
 import {RatePassengersModalModule} from './shared/components/rate-passengers-modal/rate-passengers-modal.module';
+import {MatProgressSpinnerModule} from '@angular/material';
 
 @NgModule({
 	declarations: [AppComponent],
-	imports: [
-		BrowserModule,
-		AppRoutingModule,
-		BrowserAnimationsModule,
-		FormsModule,
-		MainScreenModule,
-		MenuModule,
-		RatePassengersModalModule,
-	],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    BrowserAnimationsModule,
+    FormsModule,
+    MainScreenModule,
+    MenuModule,
+    RatePassengersModalModule,
+    MatProgressSpinnerModule,
```

Comment:
Are you sure that MatProgressSpinnerModule is required? What is it used for?

Reply:
deleted